radio/src/gui/480x272/zone.h

```diff
@@ -0,0 +1,54 @@
+
+#ifndef _ZONE_H_
+#define _ZONE_H_
+
+#include <inttypes.h>
+#include <string.h>
+
+#define LEN_ZONE_OPTION_STRING         8
+
+#define OPTION_VALUE_UNSIGNED(x)    setZoneOptionValue((uint32_t)x)
```

Comment:
Before my changes the `TextWidget::options[]` was **also** in RAM. The whole point of my change was to put it where it belongs - in flash.